unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/BuilderInWorldGodMode.cs

```diff
@@ -469,11 +470,10 @@ public void ActivateCamera(ParcelScene parcelScene)
         SetLookAtObject(parcelScene);
 
         // NOTE(Adrian): Take into account that right now to get the relative scale of the gizmos, we set the gizmos in the player position and the camera
-        Vector3 cameraPosition = DCLCharacterController.i.characterPosition.unityPosition;
-        freeCameraController.SetPosition(cameraPosition + Vector3.up * distanceEagleCamera);
-        //
-
+        Vector3 cameraPosition = DCLCharacterController.i.characterPosition.unityPosition + Vector3.up * distanceEagleCamera;
+        freeCameraController.SetPosition(cameraPosition);
         freeCameraController.LookAt(lookAtT);
+        freeCameraController.SetResetConfiguration(cameraPosition, lookAtT);
```

Comment:
suggestion: maybe it would be even more clear renaming `SetResetConfiguration` for somthing like `SetInitialConfiguration` and also setting position and lookat inside that method instead of calling them from outside?